Font tag (#815)
Codecademy/docs: content/html/concepts/tags/terms/font/font.md

---
Title: '<font>'

> Description: 'The font tag used to be used to set the font of text.'

Reviewer: ```suggestion
Description: 'Used to be used to set the font of text. This is now deprecated.'
```

> Description: 'Used to be used to set the font of text. This is now deprecated.'

Reviewer 2: ```suggestion
Description: 'Used to be used to set the font characteristics of a text. This is now deprecated.'
```

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Tags'
  - 'Style'
  - 'Typography'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---
  

> The `<font>` tag was used to set the font of text. It is not used any more in favor of the [CSS](https://www.codecademy.com/resources/docs/css) [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property instead because the `<font>` tag is now deprecated.

Reviewer: ```suggestion
The **`<font>`** tag was used to set the font of text. It is not used any more in favor of the [CSS](https://www.codecademy.com/resources/docs/css) [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property instead because the `<font>` tag is now deprecated.
```

> The **`<font>`** tag was used to set the font of text. It is not used any more in favor of the [CSS](https://www.codecademy.com/resources/docs/css) [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property instead because the `<font>` tag is now deprecated.

Reviewer 2: ```suggestion
The **`<font>`** tag was used to set the font characteristics of a text. It is now deprecated. [CSS](https://www.codecademy.com/resources/docs/css) properties like the [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property should be used instead. 
```

## Syntax

> Any combination of the following three attributes could be used in the `<font>` tag, with at least one being required:

Reviewer 2: ```suggestion
Any combination of the following three attributes could be used in the `<font>` tag, at least one attribute is required:
```

> Any combination of the following three attributes could be used in the `<font>` tag, at least one attribute is required:

Author: ```suggestion
Any combination of the following three attributes could be used in the `<font>` tag, at least any one attribute is required:
```
I feel like it may sound like to some people that 1 is required but it doesn't say it says. Should we change it to this?

Reviewer 2: sure, we could do that.

> Any combination of the following three attributes could be used in the `<font>` tag, at least any one attribute is required:

Reviewer: ```suggestion
At least one of the following three attributes is required for the `<font>`. However any combination of them can be used:
```

- `color`: What [color](https://www.codecademy.com/resources/docs/css/colors) the font/text should be. The color can be set to a named [color], hexadecimal or RGB.

> - `face`: What font the text should be. For example to have the text be `verdana` the `face` attribute would be like this `face="verdana"`

Reviewer 2: ```suggestion
- `face`: What font the text should be. For example, to have the text be `verdana` the `face` attribute would be set to `face="verdana"`
```

> - `face`: What font the text should be. For example, to have the text be `verdana` the `face` attribute would be set to `face="verdana"`.

Reviewer 2: ```suggestion
- `face`: What font the text should be. For example, to have the text be `verdana`, the `face` attribute would be set to `face="verdana"`.
```

> * `color`: What [color](https://www.codecademy.com/resources/docs/css/colors) the font/text should be. The color can be set to a named [color], hexadecimal or rgb.
> * `face`: What font the text should be. For example to have the text be `verdana` the `face` attibute would be like this `face="verdana"`
> * `size`: Sets the size of the font/text. It can use abolute units and relitive units rangeing from 1-7. Absolute units are used by `size="5"` or any other number 1-7. Relitive units are relitive the size of the `<basefont>` tag (also depricated). The size attribute would be set to `+5` or `-2`.

Reviewer: ```suggestion
- `color`: What [color](https://www.codecademy.com/resources/docs/css/colors) the font/text should be. The color can be set to a named [color], hexadecimal or rgb.
- `face`: What font the text should be. For example to have the text be `verdana` the `face` attribute would be like this `face="verdana"`
- `size`: Sets the size of the font/text. It can use absolute units and relative units ranging from 1-7. Absolute units are used by `size="5"` or any other number 1-7. Relative units are relative the size of the `<basefont>` tag (also deprecated). The size attribute would be set to `+5` or `-2`.
```
